apps/conduit-e2e/src/integration/register.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Register
+
+    As a user,
+    I want to be able to create a new account,
+    so that I can use the application for publishing articles.
+
+    Background: 
+        Given I open Register page
+
+    Scenario: Successfull registration
+        When @register I input correct username
```

Comment:
I'm not sure about this pattern, to be honest. I'm not sure if it is the best approach to add tags on every step. What do you think?

Reply:
Yes, I agree. I will remove these tags with the next PR.